command/issue.go

```diff
@@ -175,6 +175,10 @@ func issueStatus(cmd *cobra.Command, args []string) error {
 
 	out := colorableOut(cmd)
 
+	fmt.Fprint(out, utils.Gray("Relevant issues in "))
+	fmt.Fprint(out, utils.Cyan(ghrepo.FullName(baseRepo)))
```

Comment:
It's worth noting that if the repository has been renamed, but the user still hasn't updated their git remotes, this will display the pre-rename repo.

If this becomes an issue (i.e. confusing) in the future, we can use something like `resolveRemotesToRepos` from `pr create` to find the "resolved" base repo—although, that adds an additional GraphQL query.

Reply:
ah, my reverted commits glommed a repo look up onto the status commands in order to determine if the current repo was a fork. i took it out since ampinsk suggested less context was ok but it definitely showed there's only so much confidence we can have about what we're saying without a repo lookup per command. i think it'll be worth the initial performance penalty and once we do it it'll be more clear how best to cache it.